latex/Research_Strategy.tex

```diff
@@ -595,31 +595,90 @@
 
 %Aim 4 - Coordinate, run, and analyze blind challenges to advance modeling of binding (1.5 pages)
 {\bf Aim 4. Coordinate, run, and analyze blind challenges to advance modeling of binding.}
-The data collected in Aims 1-3 will drive annual SAMPL blind challenges, allowing the field to test the latest methods and force fields to assess progress, compare them against one another head-to-head, and perform sensitivity analysis to learn how much different factors (protonation state, tautomer selection, solvent model, force field, sampling method, etc.) affect predictive power. Results will then feed back into improved treatment of these factors for subsequent challenges, driving regular cycles of application, learning, and advancement.
+While the data generated in Aims 1-3 is valuable in its own right, the real impact of this work comes from using this data to conduct regular SAMPL blind challenges which test the state of the art, drive new methodological and force field innovations, allow comparative evaluation of methods, and result in downstream improvements.
+The new, progressive nature of the data generated for SAMPL6-10 means that these challenges will build on one another, and for success in later SAMPL challenges, participants must build on lessons learned from prior challenges.
+SAMPL challenges will thus serve to guide improvements to quantitative modeling of binding, facilitating regular cycles of application, learning, and improvement of models.
+
+% I didn't have this in the outline (bizarrely) but we need to briefly explain how we actually run a SAMPL challenge:
+SAMPL challenges will have submission deadlines once yearly, though not every data type or component of Aims 1-3 will be annual.
```

Comment:
Facilitate, yes, and the following comment somewhat likewise. Read down to the bottom as I address this in the last (mini) paragraph.

I don't know that we can require automation, require capturing their methods, but I certainly want to facilitate it and to the degree that we can connect up with other efforts in this space that will help. BUT, I don't think we want to say we're going to make it so people can submit their methodologies and make it so they can automate their predictions, as that's too big. Once there is a mechanism for that, we obviously want to do it, but we can't (within this grant) design the mechanism so we have to wait for it to happen. If we required it too early, it would kill participation in SAMPL which would torpedo our grant. :(